src/main/pages/portable-workspaces/factories-getting-started.adoc

```diff
@@ -8,62 +8,69 @@ folder: portable-workspaces
 ---
 
 
-A Factory is a template containing configuration to automate generating new Workspace with factory identifier added to IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
+A factory is a template containing configuration to automate generating new Workspace with factory identifier added to the IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
```

Comment:
Suggest: A factory is a template that contains the configuration to automate generation of new workspaces with the factory identifier added to the IDE URL.